src/sgame/sg_admin.cpp

```diff
@@ -5763,6 +5782,76 @@ static bool BotFillCmd( gentity_t *ent, const Cmd::Args& args )
 	return true;
 }
 
+// This command does NOT load the navmesh that it creates.
+// However the mesh will be used if you run /navgen before the first bot fill/add command.
+bool G_admin_navgen( gentity_t* ent )
+{
+	static NavmeshGenerator navgen;
+
+	const Cmd::Args& args = trap_Args();
+	if ( args.Argc() < 2 )
+	{
+		ADMP( QQ("^3navgen:^* usage: navgen (all | missing | <class>...)") );
+		return false;
+	}
+
+	std::string mapName = Cvar::GetValue( "mapname" );
+
+	std::vector<class_t> targets;
+	for ( int i = 1; i < args.Argc(); i++ )
+	{
+		if ( Str::IsIEqual( args.Argv( i ), "all" ) )
+		{
+			std::vector<class_t> all = RequiredNavmeshes();
+			targets.insert( targets.end(), all.begin(), all.end() );
```

Comment:
I considered that, but I found a usecase where it is helpful not to. If I want to make a set of navmeshes compatible with 0.52, then I can do `navgen all builder` (because the Granger one is required).